examples/data-sources/bloxone_dhcp_server/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get dhcp servers filtered by an attribute
+data "bloxone_dhcp servers" "example_by_attribute" {
+  filters = {
+    "name" = "example_dhcp_server"
+  }
+}
+
+# Get dhcp servers filtered by tag
+data "bloxone_dhcp servers" "example_by_tag" {
+  tag_filters = {
+    "region" = "eu"
+  }
+}
+
+# Get all dhcp servers
+data "bloxone_dhcp servers" "example_all" {}
```

Comment:
```suggestion
data "bloxone_dhcp_servers" "example_all" {}
```

Reply:
done

Reply:
Rename the data-source directory to the same `bloxone_dhcp_servers`

Reply:
done